build.sbt

```diff
@@ -166,22 +166,22 @@ lazy val storage = (project in file("storage"))
   .settings (
     name := "delta-storage",
     commonSettings,
-    releaseSettings, // TODO: proper artifact name
+    javaOnlyReleaseSettings,
     libraryDependencies ++= Seq(
       // User can provide any 2.x or 3.x version. We don't use any new fancy APIs. Watch out for
       // versions with known vulnerabilities.
       "org.apache.hadoop" % "hadoop-common" % "3.3.1" % "provided"
     )
   )
 
-lazy val storageDynamodb = (project in file("storage-dynamodb"))
+lazy val storageS3DynamoDB = (project in file("storage-s3-dynamodb"))
   .dependsOn(storage % "compile->compile;test->test;provided->provided")
   .dependsOn(core % "test->test")
   .settings (
-    name := "delta-storage-dynamodb",
+    name := "delta-storage-s3-dynamodb",
     commonSettings,
-    releaseSettings, // TODO: proper artifact name with no scala version
-    // Test / publishArtifact := true, // uncomment only when testing FailingDynamoDBLogStore
+    javaOnlyReleaseSettings,
+    // Test / publishArtifact := true, // uncomment only when testing FailingS3DynamoDBLogStore
```

Comment:
nit: what is this config?

Reply:
when uncommented, this will let us publish the artifact for `FailingS3DynamoDBLogStore`.
this is just like `S3DynamoDBLogStore` , but we can inject errors into it. it is used for integration tests.